.pnpmfile.cjs

```diff
@@ -258,6 +258,12 @@ function readPackage(pkg, context) {
       addPeerDependencies("asyncstorage-down", {
         "@react-native-async-storage/async-storage": "*",
       }),
+      addDependencies("@dfinity/candid", {
+        "@dfinity/principal": "*",
+      }),
+      addDependencies("@dfinity/agent", {
+        "buffer": "*",
+      }),
```

Comment:
um, you are right. This can be easily changed in the package. I hope this is not a blocker, I can make sure this be changed in the package and removed as there's a new release.

Reply:
Not a blocker :slightly_smiling_face: we can definitely do it later, I'm just afraid we'll quickly forget about it as we don't track very well things that remain to do... but it can wait for now, no problem.

Reply:
I promise, won't forget about this.